docs/release_notes/v0.11.0-alpha.md

```diff
@@ -0,0 +1,60 @@
+# Spice v0.11-alpha (Apr 15, 2024)
+
+Announcing the release of Spice.ai v0.11-alpha! 🎉
+
+The Spice.ai v0.11-alpha release brings new functionality and improves overal stability and usability.
+
+## Highlights in v0.11-alpha
+
+**New data layer architecture**: The data layer was refactored to simplify creating new data components and maximize integration with and use the power of DataFusion.
```

Comment:
What is data layer?

Reply:
Updated, thank you

The Spice.ai v0.11-alpha release brings new data connector and secret store, more control over accelerated datasets refresh functionality, better security when using MySql and Postgres and overall stability improvements.